addons/project_mrp/views/project_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="project_project_form_view_inherited" model="ir.ui.view">
+        <field name="name">project.project.view.inherited</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project" />
+        <field eval="14" name="priority"/>
```

Comment:
Do we really need the priority in this view ? 🤔

Reply:
I add the priority to keep all the buttons from different addons always shown in the same sequence

Reply:
Ok thanks. 🙂